vue/mixins/stories/locales-story.vue

```diff
@@ -0,0 +1,50 @@
+<template>

```

Comment:
Move to the stories and import the CSS if needed (e.g. https://github.com/ripe-tech/ripe-white/blob/master/vue/components/molecules/stack-navigator/stack-navigator.stories.js)